apps/ecs/index.js

```diff
@@ -122,17 +116,81 @@ module.exports = {
       ],
       next: '/original-document'
     },
-
+    '/original-document': {
+      next: '/request-check'
+    },
+    '/request-check': {
+      next: '/worker-details-1988'
+    },
+    '/worker-details-1988': {
+      next: '/worker-address'
+    },
+    '/worker-address': {
+      next: '/job-information'
+    },
     '/ongoing-appeal': {
-      next: '/check-your-answers'
+      fields: ['worker-have-ongoing-appeal'],
+      forks: [
+        {
+          target: '/request-check',
+          condition: {
+            field: 'worker-have-ongoing-appeal',
+            value: 'yes'
+          }
+        }
+      ],
+      next: '/before-1988'
     },
-    '/original-document': {
+    '/before-1988': {
+      fields: ['worker-been-in-UK-before-1988'],
+      forks: [
+        {
+          target: '/request-check-before-1988',
+          condition: {
+            field: 'worker-been-in-UK-before-1988',
+            value: 'yes'
+          }
+        }
+      ],
+      next: '/settlement-protection'
+    },
+    '/settlement-protection': {
+      fields: ['worker-applied-for-settlement-protection'],
+      next: '/request-check'
```

Comment:
could you please double check I think next step should be `\original-document`

AC2:  Given the user navigated to this page 
when they select yes or no 
then they will be navigated to the Have you seen the original document? page

Reply:
checked, the` /original-document` is the page if for arc-card. 
for settlement-protection, the target is `/request-check` when user gives answer yes or no.